src/components/TodoItem/TodoItem.css

```diff
@@ -0,0 +1,60 @@
+.list-item {
+    background-color: #dcedcd;
+    width: 90%;
+    margin-top: 20px;
+    border-radius: 10px;
+    min-height: 90px;
+    display: flex;
+    flex-direction: column;
+    word-break: break-word;
+    justify-content: space-between;
+}
+
+.list-item:hover {
+    background-color: #96cb6f;
+
+}
+
+.list-item:hover .list-item__edit-icon {
+    display: inline;
+    cursor: pointer;
+
+}
+
+.list-item__title {
+    margin: 10px 25px;
+    font-weight: 500;
+}
+
+.list-item__status-btn--new {
+    background-color: #e45a58;
+    margin: 10px 10px;
+    padding: 3px 20px;
+    border: none;
+    border-radius: 5px;
+    color: white;
+    font-weight: bold;
+    align-self: flex-end;
+}
+
+.list-item__status-btn:hover {
+    cursor: pointer;
+}
+
+.list-item__status-btn--done {
+    background-color: #3589c5;
+    margin: 10px 10px;
+    padding: 3px 20px;
+    border: none;
+    border-radius: 5px;
+    color: white;
+    font-weight: bold;
+    align-self: flex-end;
+}
+
+.list-item__edit-icon {
+    width: 20px;
+    height: 20px;
+    margin-left: 5px;
+
+}
```

Comment:
Աղասի ջան js ում ընդունված է camelCase օգտագործել
ու նաև անունների համար ոչ թե անվան մեջ պետք է ներառել թե որ դիրքում է գտնվում , այլ ինքը ով է ու ինչ սթայլ ունի 
սովորաբար ընդհանում սթայլերի համար առանձին կլաս են գրում , ու հետո ամեն մեկին տալիս են այդ կլասը , այլ ոչ թե ամեն մի կոմպոնենտի համար մի կլասով ծերից֊ծեր գրեն սթայլը

Reply:
https://medium.freecodecamp.org/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849

Reply:
Թագուհի ջան սա css nameing convention-ն է:  Սրանով եմ առաջնորդվում:

Reply:
այս հոդվածում պաճառները լավ հիմնավորված չեն 
հետագայում դու կօգտագործես նաև sass, less որտեղ արդեն մի քիչ js ոտ պետք ա լինի rule երը 
այս ընդհանուրի համար ընդունված է, որ քո առաջարկած տարբերակով գրվի գլոբալ կլասների անունները , իսկ լոկալ (օրինակ մի հատ կոմպոնենտի համար)
արդեն camelCase

այստեղ , քո մոտ բոլորը լոկալ տարբերակներ են